src/test/java/com/google/api/codegen/testdata/python_doc_types_library.baseline

```diff
@@ -0,0 +1,65 @@
+============== file: google/cloud/proto/example/library_v1/types.py ==============
```

Comment:
I would have expected this file location to be `google/cloud/library_v1/types.py`.